lib/features/pokemon_overview/pokemon_overview_page.dart

```diff
@@ -10,24 +11,40 @@ class PokemonOverviewPage extends StatelessWidget {
     Key? key,
   }) : super(key: key);
 
-  final List<Pokemon> pokemons;
+  final Async<List<Pokemon>> pokemons;
 
   @override
   Widget build(BuildContext context) {
+    void showNoPokemonAvailable(errorMessage) {
+      final snackBar = SnackBar(
+        content: Text('$errorMessage'),
```

Comment:
do you still need string interpolation here or u can already just pass the string constant 🙂